src/lib_context/recording/trace_stats_summary_pp.ml

```diff
@@ -377,6 +296,12 @@ module Table0 = struct
     let separator_row =
       [List.init (summary_name_length + 1) (Fun.const "--")]
     in
+    let dyn_summary_config_entries : [> `Key of string] list =
+      extract_all_keys summaries
+    in
```

Comment:
I think that `` `Message`` should be added to `dyn_summary_config_entries` if at least 1 summary is not parsable